requirements.txt

```diff
@@ -1,3 +1,4 @@
 aiodns<4
 aiohttp<4
 aiohttp_jinja2<2
+tox>=4.12.1
```

Comment:
AIUI, tox generally isn't included in requirements, because it's a top-level tool. It's sorta like if you added `pip` to requirements

(I'm not sure if this is true everywhere or is an openstackism)

Reply:
you're actually right
the tox requirements should be indicated dicrectly in the tox.ini configuration file under the [tox] section

Reply:
Good to know.